src/components/BooksPage.js

```diff
@@ -0,0 +1,25 @@
+import PropTypes from 'prop-types';
+import Book from './Book';
+
+const BooksPage = (props) => {
+  const { bookList } = props;
+  return (
+    <div>
+      <h1> Books </h1>
+      <ul>
+        {bookList.map((book) => <Book key={book.id} book={book} />)}
+      </ul>
+      <form id="add-book-form">
+        <input placeholder="Book Title" />
+        <input placeholder="Book Author" />
+        <button type="submit"> Add Book </button>
+      </form>
```

Comment:
- [OPTIONAL] It would be better to create a component only for your form to have your project more organized.♻️